src/components/TextBox/TextBox.jsx

```diff
@@ -12,7 +12,7 @@ const TextBox = ({ addClass, alignment, size, style, text }) => {
       {paragraphs.map((par, index) => (
         <div className={`text-box--${alignment} ${addClass || ''}`} key={index}>
           <p
-            className={`text-box text-box--${size} mb-0`}
```

Comment:
when we have multiple paragraphs, we want space between them. I initially had a conditional to remove the spacing from the last paragraph only, but felt that was too specific to our current use case. (part of the "about us" section).

in the future, this component could be used anywhere, and having the bottom margin is a good thing.